# BioHacks
BioHacks 2016

This project was based off of bioinformatics at the University of Toronto BioHacks 2016. This program allows the user to input a DNA sequence, and the program can then transcribe the sequence into mRNA and translate it into amino acids. Based off the amino acids and protein sequence, it can predict whether the structure will be alpha-helix or a beta pleated sheet by looking at the frequencies of the bases. The program then produces coordinates of where the sequence moves to create those structures and the folding nature of proteins. It does this with vector mappings by looking and comparing the distance of sulfur atoms that will create disulfide bonds. Disulfide bonds are very prominent in protein structure and cause them to fold. By producing those coordinates with vector mapping, those coordinates are then transported into a file and can be be used with a simulator to show how the protein folds and the final structure of the protein. Then a mutated DNA sequence is entered, and depending on if it's a swap, insert or delete mutation, it will undergo the exact same steps as before and be shown in the simulator. The user can then see the effects of this mutation on the DNA sequence and which functions will be lost due to this mutation.

The project was developed by first-year students who were the youngest participants at the event. The project finished 10th out of 43 groups and was deemed to be an innovative project.
